src/operator/operator_tune.cc

```diff
@@ -417,6 +417,7 @@ IMPLEMENT_BINARY_WORKLOAD_BWD(mxnet::op::mshadow_op::logical_xor);  // NOLINT()
 IMPLEMENT_BINARY_WORKLOAD_FWD_WITH_BOOL(mxnet::op::mshadow_op::bitwise_and);  // NOLINT()
 IMPLEMENT_BINARY_WORKLOAD_FWD_WITH_BOOL(mxnet::op::mshadow_op::bitwise_xor);  // NOLINT()
 IMPLEMENT_BINARY_WORKLOAD_FWD_WITH_BOOL(mxnet::op::mshadow_op::bitwise_or);  // NOLINT()
+// IMPLEMENT_BINARY_WORKLOAD_FWD(mxnet::op::mshadow_op::gamma_implicit_grad);  // NOLINT()
```

Comment:
why the comment?

Reply:
Operator tune is turned off for this OP, otherwise a floating point exception would occurred when importing MXNet. This line is now removed.